### DISCUSSION TITLE
降雨径流

### DISCUSSION THREAD

**Discussion by zhangrf21** [SynxFlow/SynxFlow#31] [Q&A] [answered]
请教一下，在debris求解器中，对于不发生侵蚀沉积的区域，雨水在地表的流动和“水+固体”混合物的流动都是遵循“曼宁摩擦+库伦摩擦”的运动阻力模型吗？单纯的水的流动是否和混合物流动有所不同？

**Comment by xiaxilin** [comment, marked as answer]
Hello, I have moved this to the discussion section. Please also ask your question in English so that it can also benefit others. 

For your question, the debris flow solver used the Manning + Mohr Columb model for the entire domain. The Mohr Columb friction is dependent on the sediment concentration, if it is 0, then the friction is purely Manning.

**Reply by zhangrf21** [reply]
Thank you very much!
